src/utils.ts

```diff
@@ -8,3 +16,10 @@ export function isAccountPrivileged(sender: string): boolean {
     sender.endsWith(':matrix.parity.io') || sender.endsWith(':web3.foundation')
   );
 }
+
+export function convertAmountToBn(amount: string): bigint {
+  const parsedAmount = Number(amount);
+  return parsedAmount < MAX_ALLOWED_DRIP_FLOAT
+    ? BigInt(parsedAmount * 10 ** decimals)
+    : BigInt(new bignum(amount).mul(bignum.pow(10, decimals)).toString());
```

Comment:
Could you add a comment explaining why this `bignum` library is needed? I don't know why you can't just use the `BigInt` directly.